File: 09 - Py-Js/ques03.js
function pyramid(rows) {
    
    console.log("Pyramid:")
    for (let i = 1; i <= rows; i++) {
        console.log(" ".repeat(rows - i) + "*".repeat(2 * i - 1))
    }

    console.log("\nReverse pyramid:")
    for (let i = rows; i >= 1; i--) {
        console.log(" ".repeat(rows - i) + "*".repeat(2 * i - 1))
    }
}
pyramid(5)

File: 09 - Py-Js/ques03.py
def pyramid(rows):

    print("pyramid:")
    for i in range(1, rows + 1):
        print(" " * (rows - i) + "*" * (2 * i - 1))
    print("\nreverse pyramid:")
    for i in range(rows, 0, -1):
        print(" " * (rows - i) + "*" * (2 * i - 1))
        
pyramid(5)